src/cursor.js

```diff
@@ -65,6 +65,16 @@ export default class Cursor {
     return content.getTagsByName(this.host, this.range, tagName)
   }
 
+  // Get all tags that are completely within the current selection.
+  getInnerTags (filterFunc) {
```

Comment:
These new functions should have a test each.

Reply:
Makes sense. The tests would go into [cursor.spec.js](https://github.com/livingdocsIO/editable.js/blob/master/spec/cursor.spec.js) file?

Reply:
Commit https://github.com/livingdocsIO/editable.js/pull/223/commits/057c3158cff780ed4f52b21d6d64c25bdb07a2e1 contains two (very) simple tests.